server/athenian/api/openapi/openapi.yaml

```diff
@@ -31,49 +31,68 @@ paths:
               schema:
```

Comment:
All the Swagger/OpenAPI tools that I've seen rely on a single file... I think it is possible though. I'll see what can be done.

Reply:
Not a big deal, don't lose your mind on that 😂

Reply:
Yep that's possible https://apihandyman.io/writing-openapi-swagger-specification-tutorial-part-8-splitting-specification-file/

Let's postpone the split until we really suffer because setting up a local CORS server for reference resolution is not very convenient in the beginning of the development...